scripts/hotspot.js

```diff
@@ -14,19 +14,23 @@
    * @param  {boolean} isSmallDeviceCB
    * @param  {H5P.ImageHotspots} parent
    */
-  ImageHotspots.Hotspot = function (config, color, id, isSmallDeviceCB, parent) {
+  ImageHotspots.Hotspot = function (config, color, id, isSmallDeviceCB, parent, index) {
     var self = this;
     this.config = config;
     this.visible = false;
     this.id = id;
     this.isSmallDeviceCB = isSmallDeviceCB;
+    this.tabIndex = index;
 
     if (!this.config.content.length) {
       throw new Error('Missing mandatory library for hotspot');
     }
 
     this.$element = $('<div/>', {
+      'id': 'hotspot-' + this.tabIndex,
```

Comment:
Might be multiple imagehotspots in the same dom, this would result in non unique IDs :/